new-packages/Core/src/Loops.candy

```diff
@@ -6,12 +6,9 @@ public builtin fun loop(body: () -> Nothing): Never
 ## TODO(marcelgarus): Implement this in pure Candy once we have tail-call elimination.
 
 impl Int {
-  public fun times(body: () -> Nothing) {
+  public fun times(body: (Int) -> Nothing) {
     # Executes the `body` `this` many times.
 
-    if(this > 0, {
-      body()
-      (this - 1).times(body)
-    })
+    0.until(this).do(action)
   }
```

Comment:
I'd undo this because it's semantically slightly different and if you really need the index, you can do `0..limit do {}`, which has the same number of chars and is more explicit.